.env.prod.example

```diff
@@ -13,12 +13,8 @@ ETH_RPC_HOST=http://localhost
 ETH_RPC_PORT=8545
 ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
 
-TYPEORM_CONNECTION = postgres
-TYPEORM_HOST = localhost
-TYPEORM_USERNAME = root
-TYPEORM_PASSWORD = root
-TYPEORM_DATABASE = anchor_db
-TYPEORM_PORT = 5432
-TYPEORM_SYNCHRONIZE = true
-TYPEORM_LOGGING = true
-TYPEORM_ENTITIES = build/models/**/*.js
+DB_HOST = localhost
+DB_USERNAME = root
```

Comment:
do you need to store `USERNAME` & `PASSWORD` in the example file? You could just parse from env connection string right?

Reply:
yes